Makefile

```diff
@@ -42,14 +42,19 @@ target-%: ## Builds the specified target defined in the Pkgfile. The build resul
 local-%: ## Builds the specified target defined in the Pkgfile using the local output type. The build result will be output to the specified local destination.
 	@$(MAKE) target-$* TARGET_ARGS="--output=type=local,dest=$(DEST) $(TARGET_ARGS)"
 
-rebuild-%: ## Builds the specified target twice into $(DEST)/build-1/2 and compares results.
-	@rm -fr $(DEST)/build-1 $(DEST)/build-2 $(DEST)/build-1.txt $(DEST)/build-2.txt
-	@docker buildx prune --all --force
-	@$(MAKE) target-$* PROGRESS=plain TARGET_ARGS="--output=type=local,dest=$(DEST)/build-1 --no-cache $(TARGET_ARGS)" 2>&1 | tee $(DEST)/build-1.txt
-	@docker buildx prune --all --force
-	@$(MAKE) target-$* PROGRESS=plain TARGET_ARGS="--output=type=local,dest=$(DEST)/build-2 --no-cache $(TARGET_ARGS)" 2>&1 | tee $(DEST)/build-2.txt
-	@find _out/ -exec touch -ch -t 202108110000 {} \;
-	@diffoscope _out/build-1 _out/build-2
+reproducibility-test:
+	@$(MAKE) reproducibility-test-local-toolchain
+
+reproducibility-test-local-%: ## Builds the specified target defined in the Pkgfile using the local output type. The build result will be output to the specified local destination.
+	@rm -rf _out1/ _out2/
+	@$(MAKE) local-$* DEST=_out1
+	@$(MAKE) local-$* DEST=_out2 TARGET_ARGS="--no-cache"
+	@touch -ch -t $$(date -d @$(SOURCE_DATE_EPOCH) +%Y%m%d0000) _out1 _out2
+	@diffoscope _out1 _out2
+	@rm -rf _out1/ _out2/
+
+foo:
```

Comment:
debug?

Reply:
was trying to parse the git log time format to unix timestamp